src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -19,21 +21,17 @@ import { FeeInformation, PriceInformation, QuoteInformationObject } from 'state/
 import { AddGpUnsupportedTokenParams } from 'state/lists/actions'
 import OperatorError, { ApiErrorCodes } from 'utils/operator/error'
 import { onlyResolvesLast } from 'utils/async'
-import { ClearQuoteParams, SetQuoteErrorParams, QuoteError } from 'state/price/actions'
 // import QuoteError, { QuoteErrorCodes, isValidQuoteError } from 'utils/operator/errors/QuoteError'
 // import { ApiErrorCodes, isValidOperatorError } from 'utils/operator/errors/OperatorError'
 import BigNumberJs from 'bignumber.js'
 import { OrderKind } from '@gnosis.pm/gp-v2-contracts'
 import { PRICE_API_TIMEOUT_MS } from 'constants/index'
 
-export interface RefetchQuoteCallbackParmams {
+export interface RefetchQuoteCallbackParams {
   quoteParams: FeeQuoteParams
   fetchFee: boolean
   previousFee?: FeeInformation
-  handlers: {
-    setLoadingCallback: () => void
-    hideLoadingCallback: () => void
-  }
+  isJustPriceRefresh: boolean
```

Comment:
nit: name is weird. isPriceRefresh should be fine. Just doesn't always mean "only"